src/Spectre.libGaClassifier/Spectre.libGaClassifier.vcxproj

```diff
@@ -132,13 +141,21 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClInclude Include="InconsistentMinimalAndMaximalFillupException.h" />
+    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="InconsistentMinimalAndMaximalFillupException.cpp" />
+    <ClInclude Include="AllLabelTypesIncludedCondition.h">
+      <SubType>Header Files</SubType>
+    </ClInclude>
+    <ClInclude Include="InconsistentMinimalAndMaximalFillupException.h" />
```

Comment:
This exception corresponds to conditions defined in `libGenetic`. I think it should go there.

Reply:
done